builder/kubernetes/builder.go

```diff
@@ -33,7 +33,7 @@ func (b *kubernetesBuilder) Build(ctx context.Context, prov provision.BuilderDep
 	if !ok {
 		return nil, errors.New("provisioner not supported")
 	}
-	if opts.BuildFromFile {
+	if opts.Dockerfile != "" {
```

Comment:
does it make sense to notify about this feature with deployV2 here (different error message)?

Reply:
This code is legacy...  it's mostly related with deploy-agent v1.